ui/apps/platform/src/Containers/SystemConfig/Form/SystemConfigForm.tsx

```diff
@@ -187,6 +215,239 @@ const SystemConfigForm = ({
 
     return (
         <Form>
```

Comment:
Ideally, I'd like to break up the Form into sections so it's easier to navigate and work with. We have around 1000 lines of code for this file now. For now, I'm displaying everything in the same file. If I find time later this sprint, I'll see if we can refactor more.

I wonder if we can have some sort of lint warning when a file gets too long so we can consider breaking it up.

Reply:
Looks like we do have a rule for that 👀